htdocs/themes/berlin_mcjc/sass/_topic.scss

```diff
@@ -0,0 +1,146 @@
+.header-background-container {
+  width: 100%;
+  height: 850px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+.topics-image {
+  background-image: url(../assets/images/Video.png);
```

Comment:
@arevalok91 could you give this a more descriptive name so we know what it is in the future?